### PR TITLE
[Revision][m]: Temporarily remove revision tag

### PR DESCRIPTION
**Pull request by risenW**
This pull removes the revision tag creation temporarily.

### CODE REVIEW

File: src/backend/filesystem.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ class FilesystemStorage extends StorageBackend {
       throw new Error(`EEXIST: file ${objectId} already exists`)
     }
     const revisionId = this._makeRevisionId()
-    metadata.revision = 0
+    // metadata.revision = 0
```
Comment (cotts) [inline src/backend/filesystem.js:28]: you can remove this commented line

File: src/backend/filesystem.js [hunk 2]
```diff
@@ -64,7 +64,7 @@ class FilesystemStorage extends StorageBackend {
     const revisionId = this._makeRevisionId()
 
     metadata = { ...currentObject, ...metadata }
-    metadata.revision = metadata.revision += 1
+    // metadata.revision = metadata.revision += 1
```
Comment (cotts) [inline src/backend/filesystem.js:67]: you can remove this also

File: src/backend/github.js [hunk 3]
```diff
@@ -73,10 +73,10 @@ class GitHubStorage extends StorageBackend {
...
 
       const { org, repoName } = this._parseId(objectId)
-      const revisionId = this._makeRevisionId()
-      const octo = this.octo
-      metadata.revision = 0
-      metadata.revisionId = revisionId
+      // const revisionId = this._makeRevisionId()
+      // const octo = this.octo
+      // metadata.revision = 0
+      // metadata.revisionId = revisionId
```
Comment (cotts) [inline src/backend/github.js:79]: remove unused commented lines

File: dist/storageBackend.js [hunk 4]
```diff
@@ -26,9 +26,9 @@ class StorageBackend {
     return (0, _uuid.v4)();
   }
 
-  _getObjectInfo(objectId, revision, author, description, metadata, createdAt) {
+  _getObjectInfo(objectId, author, description, metadata, createdAt) {
     createdAt = createdAt || new Date();
-    let objectInfo = new _types.ObjectInfo(objectId, revision, createdAt, author, description, metadata).getInfo();
+    let objectInfo = new _types.ObjectInfo(objectId, createdAt, author, description, metadata).getInfo();
```
Comment (cotts) [inline dist/storageBackend.js:31]: you can use `const` instead `let` in this case.

File: src/backend/github.js [hunk 5]
```diff
@@ -98,7 +98,7 @@ class GitHubStorage extends StorageBackend {
 
           const objectInfo = this._getObjectInfo(
             objectId,
-            revisionId,
+            // revisionId,
```
Comment (cotts) [inline src/backend/github.js:101]: ditto

File: src/backend/github.js [hunk 6]
```diff
@@ -121,11 +121,11 @@ class GitHubStorage extends StorageBackend {
       getRepo(objectId, branch, this.org, this.token)
         .then((repo) => {
           let { author, metadata, description, createdAt } = repo
-          const revisionId = metadata['revisionId'] || ''
+          // const revisionId = metadata['revisionId'] || ''
```
Comment (cotts) [inline src/backend/github.js:124]: ditto

File: src/backend/github.js [hunk 7]
```diff
@@ -121,11 +121,11 @@ class GitHubStorage extends StorageBackend {
...
 
           const objectInfo = this._getObjectInfo(
             objectId,
-            revisionId,
+            // revisionId,
```
Comment (cotts) [inline src/backend/github.js:128]: ditto

File: src/backend/github.js [hunk 8]
```diff
@@ -159,7 +159,7 @@ class GitHubStorage extends StorageBackend {
       let { org, repoName } = this._parseId(objectId)
       let existingMetadata
       let newMetadata
-      let revisionId
+      // let revisionId
```
Comment (cotts) [inline src/backend/github.js:162]: ditto

File: src/backend/github.js [hunk 9]
```diff
@@ -171,11 +171,11 @@ class GitHubStorage extends StorageBackend {
...
-          revisionId = this._makeRevisionId()
-          newMetadata.revision =
-            'revision' in existingMetadata
-              ? (existingMetadata['revision'] += 1)
-              : 0
+          // revisionId = this._makeRevisionId()
+          // newMetadata.revision =
+          //   'revision' in existingMetadata
+          //     ? (existingMetadata['revision'] += 1)
+          //     : 0
```
Comment (cotts) [inline src/backend/github.js:178]: ditto

File: src/backend/github.js [hunk 10]
```diff
@@ -197,7 +197,7 @@ class GitHubStorage extends StorageBackend {
         .then(() => {
           const objectInfo = this._getObjectInfo(
             objectId,
-            revisionId,
+            // revisionId,
```
Comment (cotts) [inline src/backend/github.js:200]: ditto

File: test/backend/filesystem.js [hunk 11]
```diff
@@ -90,6 +90,6 @@ describe('File System Backend', () => {
     let datapackage = storage.fetch(name)
     expect(datapackage.name).to.eq('India-New-2021')
     expect(datapackage.resources[0].path).to.eq('data/myresource.csv')
-    expect(datapackage.revision).to.eq(1)
+    // expect(datapackage.revision).to.eq(1)
```
Comment (cotts) [inline test/backend/filesystem.js:93]: ditto

**Review by cotts — CHANGES_REQUESTED** [review]
Awaiting changes

**Comment by risenW** [comment]
@cotts I commented them out because, we might need support later as Rufus initially stated.

**Review by cotts — APPROVED** [review]
LGTM

**Pull request merged**